optuna/visualization/_pareto_front.py

```diff
@@ -87,6 +90,8 @@ def objective(trial):
     Raises:
         :exc:`ValueError`:
             If the number of objectives of ``study`` isn't 2 or 3.
+
+    See also
```

Comment:
Ops, are these lines intentional?

Reply:
Ops. These are not what I intended. I'll fix it immediately.